script.js

```diff
@@ -23,25 +28,25 @@ document.addEventListener("blue-blocker-event", function (e) {
 	// TODO: we may want to seriously consider clearing the cache on a much less frequent
 	// cadence since we're no longer able to block users immediately and need the queue
 	ClearCache();
-	let body = null;
-	try {
-		body = JSON.parse(e.detail.body);
-		switch (e.detail.parsedUrl[1]) {
-			case "HomeLatestTimeline":
-			case "HomeTimeline":
-			case "UserTweets":
-			case "TweetDetail":
-				return HandleInstructionsResponse(e, body);
-			case "timeline/home.json":
-			case "search/adaptive.json":
-				return HandleForYou(e, body);
-			case "search/typeahead.json":
-				return HandleTypeahead(e, body);
-			default:
-				api.storage.local.set({ [EventKey]: { type: ErrorEvent, message: "found an unexpected url that we don't know how to handle", detail: e } });
+	api.storage.sync.get(DefaultOptions).then(config => {
+		try {
+			const body = JSON.parse(e.detail.body);
```

Comment:
body needs to be defined outside the `try` so that it's available in the `catch` clause on error